R/export_gtfs.R

```diff
@@ -165,7 +167,9 @@ export_gtfs <- function(gtfs,
 
   if (!as_dir) {
 
-    unlink(path, recursive = TRUE)
+    if (fs::dir_exists(path)) {
+      fs::dir_delete(path)
```

Comment:
This line is no longer covered with tests (dropping us from a clean 100% 😄). I know this is the same behavior as `unlink()` but Is there an easy way to cover it in the tests?

Reply:
@polettif Good catch - the reason that is missed is because the previous line was run, even though in all tests `path` never existed. You were just lucky that `unlink` doesn't error. Properly testing that line requires adding an extra test that calls `export_gtfs(path)` on an existing path. I'm not sure exactly where best to place this. I suggest we either:
1. First merge this, then add that test straight after; or
2. You PR to my PR to add that test.
3. Any other alternative that you might prefer?

Thanks!